src/Files.Shared/Helpers/FileExtensionHelpers.cs

```diff
@@ -144,13 +144,13 @@ public static bool IsShortcutOrUrlFile(string? filePathToCheck)
 		/// </summary>
 		/// <param name="filePathToCheck">The file path to check.</param>
 		/// <returns><c>true</c> if the filePathToCheck is an executable file; otherwise, <c>false</c>.</returns>
-		/// /// <remarks>Executable file types are; exe, bat, cmd</remarks>
+		/// /// <remarks>Executable file types are; exe, bat, cmd, py</remarks>
 		public static bool IsExecutableFile(string? filePathToCheck, bool exeOnly = false)
 		{
 			return
 				exeOnly
 					? HasExtension(filePathToCheck, ".exe")
-					: HasExtension(filePathToCheck, ".exe", ".bat", ".cmd");
+					: HasExtension(filePathToCheck, ".exe", ".bat", ".cmd", ".py");
 		}
 
```

Comment:
`IsExecutableFile` is intended for files that can be ran without installing any additional software.

Reply:
In your opinion, how should we handle python dragging an dropping, then? 

Would I have to check if python is installed first in order to allow the drop? I checked the behavior on my second windows computer without python and it does not allow the drag and drop on a python file.

Reply:
I would add a separate check for pyhon files (looks like you already did this).

Reply:
Since i had to remove python from the list of executables I had to find a different way to implement the change, let me know what you think!